scs-multiapi-gradle-plugin/build.gradle

```diff
@@ -20,7 +20,7 @@ repositories {
 }
 
 group = 'net.coru'
-version = '4.2.4'
+version = '4.2.5'
```

Comment:
Engine dependency need to be updated too

Reply:
Done